workflow/rose-suite.conf

```diff
@@ -21,11 +21,11 @@ PREPROCESSING="Lopez"  # "Lopez" or "Buckley"
 
 # Lopez Ice Floe Tracker settings
 # "Apptainer" or "Docker" or "Inject"
-IFT_INSTALL="Inject"
+IFT_INSTALL="Docker"
 # Version of the container image
-IFT_VERSION="v2.0-beta.2"
+IFT_VERSION="v2.0.0"
 # Command to run the IFT CLI if using "Inject" install mode
-IFT_COMMAND="julia --project=/workspaces/ice-floe-tracker-workspace/ice-floe-tracker-pipeline/IFTPipeline.jl /workspaces/ice-floe-tracker-workspace/ice-floe-tracker-pipeline/IFTPipeline.jl/src/cli.jl"
+IFT_COMMAND=""
```

Comment:
Is there something missing here? Why drop the full command but keep the comment?